bin/docker-build-proxy-build

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
consider implementing this script and `docker-build-proxy` in terms of a unified `docker-build-proxy-target` script:

### `docker-build-proxy`
```bash
bin/docker-build-proxy-target proxy runtime
```

### `docker-build-proxy-build`
```bash
bin/docker-build-proxy-target proxy-build build
```